timed/employment/admin.py

```diff
@@ -51,12 +51,12 @@ class SuperviseeForm(forms.ModelForm):
     class Meta:
         """Meta information for the supervisee form."""
 
-        fields = "__all__"
+        fields = "__all__"  # noqa: DJ007
         model = models.User.supervisors.through
 
 
 class SupervisorInline(admin.TabularInline):
-    autocomplete_fields = ["to_user"]
+    autocomplete_fields = ("to_user",)
```

Comment:
since i disabled `RUF012` it wouldn't complain about a list

Reply:
been reading up on [RUF012](https://docs.astral.sh/ruff/rules/mutable-class-default/) do we have any mutability requirements on all of these class vars?

Reply:
oh my, the [docs](https://docs.djangoproject.com/en/5.0/ref/contrib/admin/) are unclear... for autocomplete_fields it's a list, but other fields like readonly_fields can be both.

Reply:
and, for autocomplete_fields, it looks like the default implementation returns `()` as a default but calls it a list in docs

so what we really will want this to be is `tuple(str, ...)`. this would make it immutable, which is a good thing, and also makes it clear that it is not a tuple containing variable types

it seems that typeddjango [agrees with my analysis](https://github.com/typeddjango/django-stubs/blob/e11955f05994e196dcee004811ce2e05ff230462/django-stubs/contrib/admin/options.pyi#L89) allowing both tuples and lists for the autocomplete_fields case, hence a `tuple(str, ...)` being the mostest correctest